VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -509,4 +509,13 @@
   <data name="SwitchDoesNotHandleAllEnumOptionsCodeFixTitle" xml:space="preserve">
     <value>Add case(s).</value>
   </data>
+  <data name="EqualsAndGetHashcodeNotImplementedTogetherAnalyzerMessage" xml:space="preserve">
+    <value>Both Equals() and GetHashcode() must be implemented.</value>
```

Comment:
I would reword this to "Equals() and GetHashcode() must be implemented together" to stress that that discrepancy is the problem